controllers/insights/views/insights-documents.njk

```diff
@@ -0,0 +1,164 @@
+{% extends "layouts/main.njk" %}
+{% from "components/hero.njk" import hero with context %}
+{% from "components/content-box/macro.njk" import contentBox %}
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/article-teaser/macro.njk" import articleTeaser %}
+{% from "components/split-nav/macro.njk" import splitNav %}
+{% from "components/card/macro.njk" import card %}
+
+{% from "../../updates/views/view-helpers.njk" import entryRegions with context %}
```

Comment:
This feels a bit ugly – do you think we need a more general shared helpers file/folder?

Reply:
I'd say that `views/components` is exactly for this 😄 . If we need to use this across routers can this become a component?

Reply:
Funnily enough I ended up not using this component here, but I did move `facetGroup` to its own component as it's now shared between here and past grants search.